src/modules/partition/gui/CreatePartitionDialog.cpp

```diff
@@ -52,6 +55,14 @@ CreatePartitionDialog::CreatePartitionDialog( Device* device, PartitionNode* par
 {
     m_ui->setupUi( this );
 
+    QStringList mountPoints;
+    mountPoints << "/" << "/boot" << "/home" << "/opt" << "/usr" << "/var";
```

Comment:
It would be better to use a C++11 initialization list here instead of appending.
Like this: `QStringList mountPoints = { "/", "/boot",... };`